src/state/types.ts

```diff
@@ -58,8 +58,24 @@ export interface FarmsState {
   userDataLoaded: boolean
 }
 
+export interface CakeVault {
+  // When the balance strings are typed as BigNumbers, it throws an error after serialising the BigNumber with .toJSON(): Type 'string' is not assignable to type 'BigNumber'
+  totalShares?: string
```

Comment:
When the balance strings are typed as BigNumbers, it throws an error after serialising the BigNumber with .toJSON():

```
Type 'string' is not assignable to type 'BigNumber'
```

Hence typing them as strings within redux.